.github/workflows/stale_issues.yml

```diff
@@ -0,0 +1,20 @@
+name: Mark issues as stale
+
+on:
+  schedule:
+  - cron: "0 0 * * *"
+
+jobs:
+  stale:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/stale@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-issue-message: "This issue has been inactive for long enough to be automatically marked as stale. If this was a bug report and hasn't been addressed yet, and is still a probelm, please don't hesitate to notify a maintainer."
+        stale-issue-label: 'Stale'
+        exempt-issue-label: 'Triaged'
+        days-before-stale: 30
+        days-before-close: 180
```

Comment:
Decide if you want this to actually close issues or not. It's handy for us, but it might not be wanted for you all. 180 days is long enough to where anything not marked as `triaged` is likely being ignored anyways.